src/plugins/data/public/search/aggs/buckets/create_filter/histogram.ts

```diff
@@ -19,15 +19,20 @@
 
 import { IBucketAggConfig } from '../bucket_agg_type';
 import { buildRangeFilter, RangeFilterParams } from '../../../../../common';
+import { GetInternalStartServicesFn } from '../../../../types';
 
-export const createFilterHistogram = (aggConfig: IBucketAggConfig, key: string) => {
-  const value = parseInt(key, 10);
-  const params: RangeFilterParams = { gte: value, lt: value + aggConfig.params.interval };
+/** @internal */
+export const createFilterHistogram = (getInternalStartServices: GetInternalStartServicesFn) => {
```

Comment:
Now that we don't have `aggConfig.fieldFormatter`, these need to be wrapped in a provider function so that `getInternalStartServices` can be passed in to access `fieldFormats.deserialize` instead. (This is also one of the reasons agg types still depend on field formats, even though agg configs & agg config doesn't)